environment.md

```diff
@@ -59,23 +75,16 @@ The priority of the transports are Direct file IO, SSH2, FTP PHP Extension, FTP
 
 ## Database
 
-WordPress stores content, configuration, and other information inside of a database. The database for a WordPress website is where all of the WordPress website's user-defined data is stored. WordPress is primarily designed to use a MySQL or MySQL-related database server; WordPress officially only supports MySQL or MariaDB, a drop-in replacement for MySQL.
-
-### MySQL
-
-MySQL is a widely used relational database server. MySQL comes in both open-source and commercial distributions. Either distribution should work with MySQL. The commercial distribution has additional features not found in the open-source distribution; however, WordPress does not require or use the additional features. It is designed to run on either the open-source or the commercial distribution.
-
-### MariaDB
-
-MariaDB is a drop-in replacement for MySQL supported by WordPress. It's a fork of the open-source distribution of MySQL. MariaDB was originally created to maintain a more open-source version of MySQL, but it has grown into its own relational database server alternative to MySQL with features and changes not found in MySQL. Despite its differences, MariaDB is still a fully compatible replacement for MySQL and can generally seamlessly replace MySQL.
+For data storage, WordPress uses systems based on MySQL distributions, such as MySQL, MariaDB or Percona. All three systems are compatible with WordPress, so at this moment any of the three can be used for its operation.
 
-### Percona
+The use of these versions is recommended, both for performance and security reasons, although previous versions usually work without problems:
 
-Percona server is a drop-in replacement for MySQL, focused on performance. Although it's a drop-in replacement for MySQL, WordPress does not officially support Percona.
+* [MariaDB](https://mariadb.org/) 10.4.x / 10.3.x / 10.2.x
+* [MySQL](https://dev.mysql.com/downloads/mysql/) 8.0.x / 5.7.x
+* [Percona MySQL Server](https://www.percona.com/software/mysql-database/percona-server) 8.0.x / 5.7.x
 
-### Recommended Versions
+## How do I know which version I have?
 
-WordPress recommends the following versions for your Database:
+By choosing a web server, PHP and a database you will get your WordPress base installation working at first.
```

Comment:
I don't think I completely understand what you mean. "Choosing a web server, PHP and database server should be enough to get a WordPress install set up.", maybe?

I like a transition sentence. In practice, I've sometimes had issues with servers where a lot of other micro-packages are missing with distros, and they can be difficult to configure from scratch, and I wonder if maybe this over simplifies setup a little bit. 

Unless I'm missing context (totally possible), maybe it'd work to skip this sentence and start the section with the next part?
"If WordPress is already installed and you want to know what it's using, ..."

Reply:
Deleted... it has no sense :P

Reply:
👍